hashtag-actor/Actors/HashTagActor.cs

```diff
@@ -62,6 +62,7 @@ public async Task<int> GetCount(string hashtagAndSentiment)
             try
             {
                 count = await this.StateManager.GetStateAsync<int>(hashtagAndSentiment);
+                Console.WriteLine($"GetCount for {hashtagAndSentiment} found and it is {count}.");
```

Comment:
We init a logger on line 36, can we use it instead of Console? I know we use Console other places but it feels silly to have a logger and not use it.

Reply:
Just updated it to use a logger instead in this whole file.

(Are we using DI to create loggers anywhere in this project?)

Reply:
We appear to be inconsistent across the apps. So another thing to add to the backlog of best practices for the longhauls 😄